rj_gameplay/rj_gameplay/play/basic_defense.py

```diff
@@ -2,9 +2,8 @@
 
 from rj_gameplay.tactic import wall_tactic, goalie_tactic, nmark_tactic
 import stp.role
-from stp.role.assignment.naive import NaiveRoleAssignment
 import stp.rc
-from typing import Dict, List, Tuple, Type
+from typing import Dict, List, Type  # noqa: F401
```

Comment:
If one of these imports is unused, just delete it rather than putting the noqa flag

Reply:
ok